packages/core/src/readers/LlamaParseReader.ts

```diff
@@ -155,9 +155,12 @@ export class LlamaParseReader extends FileReader {
   private async createJob(data: Buffer): Promise<string> {
     // Load data, set the mime type
     const mimeType = await this.getMimeType(data);
+    const matchingExtension = Object.keys(SupportedFiles).filter(
```

Comment:
should we start the job if the extension is not matching?

Reply:
This is just supposed to enhance readability by logging the file extensions instead of the mime type.
The getMimeType method should throw an error beforehand anyway if the file is not supported, so a non matching extensions, should have no influence on the job itself

Reply:
I see, alternatively you could return the extension as well from `getMimeType`?

Reply:
That works and actually fixes a potential mismatch error between most likely and assigned mime type. Thanks!